docs/source/AdministratorGuide/Systems/DataManagement/fts3.rst

```diff
@@ -154,4 +154,49 @@ FTS3 Plugins
     The ``FTS3Plugin`` option
 
 
-The ``FTS3Plugin`` option allows one to specify a plugin to alter some default choices made by the FTS3 system. These choices concern the list of third party protocols used, the selection of a source storage element as well as the FTS activity used. This can be useful if you want to implement a matrix-like selection of protocols, or if some links require specific protocols, etc. The plugins must be placed in :py:mod:`DIRAC.DataManagementSystem.private.FTS3Plugins`. The default behaviors, as well as the documentation on how to implement your own plugin can be found in :py:mod:`DIRAC.DataManagementSystem.private.FTS3Plugins.DefaultFTS3Plugin`
+The ``FTS3Plugin`` option allows one to specify a plugin to alter some default choices made by the FTS3 system. These choices concern:
+
+   * the list of third party protocols used
+   * the selection of a source storage element
+   * the FTS activity used
+   * The multihop strategy
+
+This can be useful if you want to implement a matrix-like selection of protocols, or if some links require specific protocols, etc. The plugins must be placed in :py:mod:`DIRAC.DataManagementSystem.private.FTS3Plugins`. The default behaviors, as well as the documentation on how to implement your own plugin can be found in :py:mod:`DIRAC.DataManagementSystem.private.FTS3Plugins.DefaultFTS3Plugin`
+
+
+MultiHop support
+----------------
+
+.. versionadded:: v7r3p21
+
+.. |trade|   unicode:: U+2122
+
+.. warning::
+   Experimental feature with drawbacks, see below
+
+In the 2020s, the grid world made a leap forward as hardly ever before. In order to make the best use of all the years of experience in manipulating common protocols and fully using the dedicated network to interconnect all sites and storages, WLCG blessed the following ThirdPartyCopy strategy for the years to come:
+
+* incompatible protocols between storages are okay
+* storages not reachable from other site storages are okay
+
+They named this strategy *MultiHop* |trade|. It consists of turning a transfer `` A -> B`` into ``A -> I1 -> I2 -> ... -> In -> B``.
```

Comment:
```suggestion
They named this strategy *MultiHop* |trade|. It consists of turning a transfer ``A -> B`` into ``A -> I1 -> I2 -> ... -> In -> B``.
```
The space breaks rendering: https://dirac--5809.org.readthedocs.build/en/5809/AdministratorGuide/Systems/DataManagement/fts3.html#multihop-support

Reply:
thanks !